geode-core/src/distributedTest/java/org/apache/geode/cache30/DistributedAckRegionCCEDUnitTest.java

```
@@ -147,129 +146,118 @@ public void testConcurrentOpWithGII() {
-        } catch (UnknownHostException e) {
-          fail("could not create member id: ", e);
+          Thread.sleep(1000);
+        } catch (InterruptedException e) {
+          return;
         }
+      }
+      assertThat(adv.adviseGeneric())
+          .withFailMessage("other member never came on line")
+          .isNotEmpty();
+
+      DistributedCacheOperation.LOSS_SIMULATION_RATIO = 200.0; // inhibit all messaging
+      try {
+        CCRegion.put("mykey", "initialValue");
+        CCRegion.destroy("mykey");
+      } finally {
+        DistributedCacheOperation.LOSS_SIMULATION_RATIO = 0.0;
+      }
 
-        // generate an event to distribute that contains the fake version tag
-        EntryEventImpl event =
-            EntryEventImpl.create(CCRegion, Operation.UPDATE, key, false, mbr, true, false);
-        event.setNewValue("newValue");
-        event.setVersionTag(tag);
-
-        // this should update the controller's cache with the updated value but leave this cache
-        // alone
-        DistributedCacheOperation op = new UpdateOperation(event, tag.getVersionTimeStamp());
-        op.distribute();
-        event.release();
+      // generate a fake version tag for the message
+      final VersionStamp versionStamp = CCRegion.getRegionEntry(key).getVersionStamp();
+      VersionTag<InternalDistributedMember> tag =
+          (VersionTag<InternalDistributedMember>) versionStamp.asVersionTag();
+      // create a fake member ID that will be < mine and lose a concurrency check
+      MemberData nm =
+          CCRegion.getDistributionManager().getDistributionManagerId().getMemberData();
+      InternalDistributedMember mbr = null;
+      try {
+        mbr = new InternalDistributedMember(nm.getInetAddress().getCanonicalHostName(),
+            nm.getMembershipPort() - 1, "fake_id", "fake_id_ustring",
+            ClusterDistributionManager.NORMAL_DM_TYPE, null, null);
+        tag.setMemberID(mbr);
+      } catch (UnknownHostException e) {
```

Delete any `catch` block with a `fail` and let the test throw `UnknownHostException`.

---

done